frontend/app/routes/$lang/_protected/access-to-governmental-benefits/view.tsx

```diff
@@ -100,93 +100,67 @@ export default function AccessToGovernmentalsBenefitsView() {
-          >
-            {t('access-to-governmental-benefits:access-to-governmental-benefits.view.update-your-access-text')}
-          </InlineLink>
-        </div>
-
-        <fetcher.Form method="post" noValidate>
-          <input type="hidden" name="_csrf" value={csrfToken} />
-          <div className="flex flex-wrap items-center gap-3">
-            <ButtonLink id="back-button" routeId="$lang/_protected/access-to-governmental-benefits/index" params={params} data-gc-analytics-customclick="ESDC-EDSC:CDCP Personal Information:Back - Access to governmental benefits click">
-              {t('access-to-governmental-benefits:access-to-governmental-benefits.view.back')}
-            </ButtonLink>
+  return (
+    <div className="max-w-prose">
+      {updatedInfo && (
+        <ContextualAlert type="success">
+          <h2 className="text-xl font-semibold">{t('access-to-governmental-benefits:access-to-governmental-benefits.view.info-updated.your-info-has-been-updated')}</h2>
+        </ContextualAlert>
+      )}
+      {hasDentalPlans ? (
+        <>
+          <div className="mb-5 space-y-4">
+            {personalInformation.provincialTerritorialDentalPlanId ? (
+              <div className="mb-5 space-y-4">
+                <h2 className="font-bold"> {t('access-to-governmental-benefits:access-to-governmental-benefits.view.provincial-or-territorial-dental-benefit')}</h2>
+                <ul className="list-disc space-y-6 pl-7">
+                  <li>{provincialAndTerritorialProgramName}</li>
+                </ul>
+              </div>
+            ) : (
+              <div className="mb-5 space-y-4">
+                <h2 className="font-bold"> {t('access-to-governmental-benefits:access-to-governmental-benefits.view.provincial-or-territorial-dental-benefit')}</h2>
+              </div>
+            )}
           </div>
-        </fetcher.Form>
-      </div>
-    );
-  } else {
-    return (
-      <div className="max-w-prose">
-        {updatedInfo && (
-          <ContextualAlert type="success">
-            <h2 className="text-xl font-semibold">{t('access-to-governmental-benefits:access-to-governmental-benefits.view.info-updated.your-info-has-been-updated')}</h2>
-          </ContextualAlert>
-        )}
+          <div className="mb-5 space-y-4">
+            {personalInformation.federalDentalPlanId ? (
+              <div className="mb-5 space-y-4">
+                <h2 className="font-bold"> {t('access-to-governmental-benefits:access-to-governmental-benefits.view.federal-benefits-dental-benefits')}</h2>
+
+                <ul className="list-disc space-y-6 pl-7">
+                  <li>{federalSocialProgramName} </li>
+                </ul>
+              </div>
+            ) : (
+              <div className="mb-5 space-y-4">
+                <h2 className="font-bold"> {t('access-to-governmental-benefits:access-to-governmental-benefits.view.federal-benefits-dental-benefits')}</h2>
+              </div>
+            )}
```

Comment:
We don't want to be displaying a heading with no text below (not sure if that's how the page renders, when I ran this branch I received a 500 page when attempting to hit this route).

```suggestion
            {personalInformation.federalDentalPlanId && (
              <div className="mb-5 space-y-4">
                <h2 className="font-bold"> {t('access-to-governmental-benefits:access-to-governmental-benefits.view.federal-benefits-dental-benefits')}</h2>

                <ul className="list-disc space-y-6 pl-7">
                  <li>{federalSocialProgramName} </li>
                </ul>
              </div>
            )}
```

(Same comment can be applied to `provincialTerritorialDentalPlanId` check above).

Reply:
Sure, makes sense. Implemented in commit da8ead45e49925194da4aab17a9f4077778f684d.